README.md

```diff
@@ -64,25 +69,40 @@ Selectors aren't limited to tag names, though (otherwise why not just use `docum
 </body>
 ```
 
-In the above example, the first query says, "Starting from `document` (the object we've called `querySelector()` on), find a `ul` with a `className` of `ranked-list` (the `.` is for `className`). Then find an `li` that is a child of that `ul`. Then find a `ul` that is a child (but not necessarily a direct descendant) of that `li`. Finally, find an `li` that is a child of that (second) `ul`."
+In the above example, the first query says, "Starting from `document` (the
+object we've called `querySelector()` on), find a `ul` with a `className` of
+`ranked-list` (the `.` is for `className`). Then find an `li` that is a child of
+that `ul`. Then find a `ul` that is a child (but not necessarily a direct
+descendant) of that `li`. Finally, find an `li` that is a child of that (second)
+`ul`."
 
-**NOTE**: The HTML property `class` is referred to as `className` in JavaScript. It's... unfortunate.
+**NOTE:*** The HTML property `class` is referred to as `className` in JavaScript.
+It's... unfortunate.
 
-What, then, does the second call to `querySelector()` say? Puzzle it out for a bit, and then read on.
+What, then, does the second call to `querySelector()` say? Puzzle it out for a
+bit, and then read on.
 
 Puzzle a bit longer!
 
 Just a bit longer!
 
-Okay, the second call says, "Starting from `document`, find a `ul` with a `className` of `unranked-list`. Then find an `li` descended from `ul.unranked-list` and a `div` descended from that `li`."
+Okay, the second call says, "Starting from `document`, find a `ul` with a
+`className` of `unranked-list`. Then find an `li` descended from `ul.unranked-
+list` and a `div` descended from that `li`."
 
 ### Interlude: Selectors
 
-Now is probably a good time to read up on [selectors](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Getting_Started/Selectors). They're super important and relatively straightforward to pick up. Play around on the MDN page while you're getting the hang of it! Then come back when you're ready.
+Now is probably a good time to brush up on
+[selectors](https://developer.mozilla.org/en-
+US/docs/Web/Guide/CSS/Getting_Started/Selectors). Play around on the MDN page,
+then come back when you're ready.
 
 ### `querySelectorAll()`
 
-`querySelectorAll` works a lot like `querySelector()` -- it accepts a selector as its argument, and it searches starting from the element that it's called on (or from `document`) -- but instead of returning the _first_ match, it returns a NodeList (which, remember, is _not_ an Array) of matching elements.
+`querySelectorAll` works a lot like `querySelector()` -- it accepts a selector
+as its argument, and it searches starting from the element that it's called on
+(or from `document`) -- but instead of returning the _first_ match, it returns a
+NodeList (which, remember, is _not_ an Array) of matching elements.
```

Comment:
...of *all* matching elements.

Use to contrast _first_ match in previous clause